Nos/Views/NoteComposer/ComposerActionBar.swift

```diff
@@ -134,7 +84,102 @@ struct ComposerActionBar: View {
             alignment: .top
         )
     }
-    
+
+    var defaultMenu: some View {
+        HStack(spacing: 0) {
+            if !showPreview {
+                attachMediaView
+                expirationTimeView
+            }
+            Spacer()
+            previewToggleView
+        }
+    }
+
+    /// Attach Media View
+    ///
+    /// Used by ``ComposerActionBar/defaultMenu``.
```

Comment:
I don't think the "Used by" line adds any value -- if it was used by a separate type in a separate file, it would be more useful. This is used by code that's less than 10 lines above so 🤷 
```suggestion
```